crates/rollup-boost/src/cli.rs

```diff
@@ -90,9 +90,7 @@ pub struct Args {
 
 impl Args {
     pub async fn run(self) -> eyre::Result<()> {
-        rustls::crypto::ring::default_provider()
-            .install_default()
-            .expect("Failed to install TLS ring CryptoProvider");
+        let _ = rustls::crypto::ring::default_provider().install_default();
```

Comment:
Same as with the other changes. It fails if you try to install twice.

Reply:
would have a warning log here instead of a silent failure